### PR TITLE
Fix ExprPlain always getting the base material from ItemTypes

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Calls ItemType#getRandom() to ensure the item is a accurate representation of the alias (e.g. any sword) before getting the material.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** #5994

### CODE REVIEW

**Review by AyhamAl-Ali — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]

**Comment by Pikachu920** [comment]
can we add a regression test please?

**Comment by sovdeeth** [comment]
> can we add a regression test please?

I didn't know how to do one that didn't have an inherently random chance of failing. Any advice?

**Review by Moderocky — APPROVED** [review]

**Comment by Pikachu920** [comment]
> > can we add a regression test please?
> 
> I didn't know how to do one that didn't have an inherently random chance of failing. Any advice?

i think you could use mockito to ensure getRandom gets called on the itemtype

File: src/test/java/org/skriptlang/skript/test/tests/regression/ExprPlainAliasTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import ch.njol.skript.test.runner.SkriptJUnitTest;
+import ch.njol.skript.variables.Variables;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.easymock.EasyMock;
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
```
Comment (Pikachu920) [inline src/test/java/org/skriptlang/skript/test/tests/regression/ExprPlainAliasTest.java:34]:
```suggestion
// https://github.com/SkriptLang/Skript/issues/5994
```

**Review by Pikachu920 — CHANGES_REQUESTED** [review]

**Review by Pikachu920 — APPROVED** [review]

**Pull request merged**